docs/tutorials/basic/module.md

```diff
@@ -78,14 +78,70 @@ mod = mx.mod.Module(symbol=net,
                     label_names=['softmax_label'])
 ```
 
+## Intermediate-level Interface
+
+We have created module. Now we can use module's intermediate-level APIs to do
```

Comment:
Now let us see how to run training and inference using module's intermediate-level APIs.